test/OpenTelemetry.Tests/Impl/Trace/Export/BatchingSpanProcessorTests.cs

```diff
@@ -146,9 +147,11 @@ public void ExporterIsSlowerThanDelay()
                 spans.Add(CreateSampledEndedSpan(i.ToString()));
             }
 
-            var exported = WaitForSpans(spanExporter, 20, TimeSpan.FromSeconds(1.1));
+            var exported = WaitForSpans(spanExporter, 20, TimeSpan.FromSeconds(2));
```

Comment:
instead of magic number 2, maybe use DefaultTimeout * 2.

Reply:
* 2 t will be magic too.